### DISCUSSION TITLE
I don't see any drive routes graphics on maps

### DISCUSSION THREAD

**Discussion by singalsu** [teslamate-org/teslamate#2000] [Getting started]
I've set up Teslamate to Ubuntu 20.04 computer natively without container by following the manual install instructions. Grafana panels look sane except that my drive routes do not appea to maps. In Teslamate (port 4000) web page the car location in map is shown correctly on map. But not in Grafana panels.

If I click on some curves the map returns to max zoom out level if I had zoomed to area where the route should have been. I may well be missing how route graphics are activated or shown (what kind of mouse clicks expected) . 

In Charging Stats dashboard there's a red ! triangle message "Panel plugin not found: grafana-map-panel"

I'm not familiar with containers. Would some container build script show the needed grafana plugins and proper versions.

**Comment by singalsu** [comment]
The route graphs are still missing but I was able to sort out other issues, hope so.

The plugin versions and install commands can be seen from grafana/Dockerfile for checked out version.

The not shown widgets were due to two unsigned plugins. Editing line to /etc/grafana.ini section plugins helped

`allow_loading_unsigned_plugins = grafana-map-panel,natel-discrete-panel`

Help with missing routes would be great since I can't find any errors from logs about it.

**Comment by singalsu** [comment]
Trackmap works after I downgraded grafana version `sudo apt-get install grafana=7.5.10`. I also installed alexandra version of trackmap `grafana-cli --pluginUrl https://github.com/alexandrainst/alexandra-trackmap-panel/archive/refs/heads/master.zip plugins install alexandra-trackmap-panel` but I'm not sure it was needed. This version needed unsigned plugin load allow.

**Reply by withanhdammit** [reply]
Thanks!  I had issues with the trackmap as well.  Although I installed grafana 7.5.11 to fix the issue based on your suggestion.

**Reply by folofjc** [reply]
Yep, I had the same problem. Was on 8.1.3 and never saw anything on the maps. Downgraded to 7.5.10 and the tracks showed up on the maps. Thanks!

**Reply by edasque** [reply]
On 8.x did anyone notice Js errors in the console that could point to the issue?

**Comment by singalsu** [comment]
To prevent upgrades to change grafana version you can do

`sudo echo "grafana hold" | sudo dpkg --set-selections`

To check the status and remove hold, do

```
dpkg --get-selections grafana
`sudo echo "grafana install" | sudo dpkg --set-selections`
```
